keps/sig-apps/3017-pod-healthy-policy-for-pdb/kep.yaml

```diff
@@ -26,13 +26,13 @@ replaces:
 
 stage: stable
 
-latest-milestone: "v1.30"
+latest-milestone: "v1.31"
```

Comment:
One nit above, can you update this line:
```
status: implementable
```
to 
```
status: implemented
```

Reply:
+1, updated